addons/mail/i18n/mail.pot

```diff
@@ -7664,6 +7664,12 @@ msgid ""
 "from this template"
```

Comment:
Maybe complement a bit the commit message to explain the heuristic we used to allow attachment deletion?